hummingbot/model/market_data.py

```diff
@@ -9,16 +9,20 @@
 class MarketData(HummingbotBase):
     __tablename__ = "MarketData"
     __table_args__ = (
-        Index("timestamp", "exchange", "trading_pair"),
+        PrimaryKeyConstraint("timestamp", "exchange", "trading_pair"),
+        Index("idx_market_data_timestamp", "timestamp"),
+        Index("idx_market_data_trading_pair", "trading_pair"),
     )
 
-    timestamp = Column(SqliteDecimal(6), primary_key=True, nullable=False)
+    timestamp = Column(SqliteDecimal(6), nullable=False)  # Milliseconds since epoch: int(time.time() * 1e3)
     exchange = Column(Text, nullable=False)
     trading_pair = Column(Text, nullable=False)
     mid_price = Column(SqliteDecimal(6), nullable=False)
     best_bid = Column(SqliteDecimal(6), nullable=False)
     best_ask = Column(SqliteDecimal(6), nullable=False)
-    order_book = Column(JSON)
+    spread = Column(SqliteDecimal(6), nullable=True)  # Absolute spread: ask - bid
+    spread_pct = Column(SqliteDecimal(6), nullable=True)  # Spread percentage: (spread / mid_price) * 100
```

Comment:
```suggestion
    spread = Column(SqliteDecimal(6), nullable=True)  # Spread percentage: (ask - bid / mid_price) * 100
```
spread is not relevant data, so let's not store that